.github/workflows/scripts/scanner-update-offline-bundle.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -eu
+
+SCANNER_V4_DEFS_BUCKET="https://storage.googleapis.com/scanner-v4-test"
+ROX_PRODUCT_VERSION="$1"
+PRODUCT_VERSION="${ROX_PRODUCT_VERSION}.0"
+declare -A files_to_download=(
+    ["v4/vulns.json.zst"]="${SCANNER_V4_DEFS_BUCKET}/vulnerability-bundles/${PRODUCT_VERSION}/vulns.json.zst"
+    ["v4/repository-to-cpe.json"]="${SCANNER_V4_DEFS_BUCKET}/redhat-repository-mappings/repository-to-cpe.json"
+    ["v4/container-name-repos-map.json"]="${SCANNER_V4_DEFS_BUCKET}/redhat-repository-mappings/container-name-repos-map.json"
+    ["v2/scanner-vuln-updates.zip"]="https://storage.googleapis.com/scanner-support-public/offline/v1/scanner-vuln-updates.zip"
```

Comment:
I see we are downloading the current the V2 latest bundle as part of this job.  What is the plan for updating the `latest` bundle to include the Scanner V4 stuff as well? that in a separate PR?

Reply:
Updated to `unzip -n v2/scanner-vuln-updates.zip scanner-defs.zip k8s-istio.zip -d $dir`
That should extract desired files from the zip

Reply:
I added `gsutil cp "$dir/scanner-v4-defs-${ROX_PRODUCT_VERSION}.zip" "gs://scanner-v4-test/offline-bundles/"`
So `Job A` will only depend on scanner-v4-defs file but not **_scanner-vulns-${ROX_PRODUCT_VERSION}.zip_**